README.md

```diff
@@ -122,7 +118,7 @@ Relative path to a webpack config specific to the server bundle.
 
 ```sh
 npm run test
-npm run test:watch # this is to watch your tests
+npm run test -- --watch # this is to watch your tests
```

Comment:
What level of experience/cut'n'pasta are we expecting?
This looks _mandatory_ judging by the title if we expect low experienced users. Find for users with more experience